app/src/main/res/layout/multiple_choice_interaction_items.xml

```diff
@@ -17,7 +17,8 @@
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:minHeight="48dp"
-    android:layout_margin="4dp"
+    android:paddingTop="8dp"
```

Comment:
**Comment 3**

If you see mocks, between two items the gap is of `24dp` which means we can consider that as 12dp for above item and 12dp for below item. 
Now as per comment 2, where we have mentioned `4dp` top/bottom padding, we are mentioning remaining `8dp` as top and bottom here.